src/templates/calendar/calendar.js

```diff
@@ -0,0 +1,136 @@
+import { React, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import dayjs from 'dayjs';
+import localeDa from 'dayjs/locale/da'; // With a custom alias for the locale object
+import localizedFormat from 'dayjs/plugin/localizedFormat';
+import { IntlProvider, FormattedMessage } from 'react-intl';
+import './calendar.scss';
+
+/**
+ * Slideshow component.
+ *
+ * @param {object} props
+ *   Props.
+ * @param {object} props.content
+ *   The slide content.
+ * @returns {JSX.Element}
+ *   The component.
+ */
+function Calendar({ content }) {
+  dayjs.extend(localizedFormat);
+  const [translations, setTranslations] = useState();
+  const { backgroundColor, hasDateAndTime, title, events } = content;
+  const classes = `template-calendar ${backgroundColor}`;
+
+  /**
+   * Imports language strings.
+   */
+  useEffect(() => {
+    import('./lang/da.json').then((data) => {
+      setTranslations(data);
+    });
+  }, []);
+
+  // Sort events by datetime and filter away events that are done.
+  const sortedEvents = events
+    .filter((e) => {
+      return new Date(e.datetime).getTime() > new Date().getTime();
+    })
+    .sort((a, b) => a.datetime.localeCompare(b.datetime));
+
+  /**
+   * Creates and updates the datestring.
+   *
+   * @returns {string}
+   *    Returns an updated datestring.
+   */
+  function useNewTimer() {
+    /**
+     * Capitalize the datestring, as it starts with the weekday.
+     *
+     * @param {string} s
+     *    The string to capitalize.
+     * @returns {string}
+     *    The capitalized string.
+     */
+    const capitalize = (s) => {
+      return s.charAt(0).toUpperCase() + s.slice(1);
+    };
+
+    const [date, setDate] = useState();
+
+    /**
+     * Updates the date string.
+     */
+    const displayTime = () => {
+      setDate(capitalize(dayjs().locale(localeDa).format('LLLL')));
+    };
+
+    /**
+     * Has timer, that calls displaytime to update date.
+     */
+    useEffect(() => {
+      displayTime();
+      const timer = setInterval(() => displayTime(), 1000);
+      return function cleanup() {
+        clearInterval(timer);
+      };
+    });
+
+    return date;
+  }
+
+  const date = hasDateAndTime ? useNewTimer() : undefined;
```

Comment:
is useNewTimer called for each render?
It should only be setup on mount and removed at unmount